tests/components/dsmr/test_config_flow.py

```diff
@@ -414,6 +421,7 @@ async def test_setup_serial_fail(com_mock, hass, dsmr_connection_send_validate_f
             result["flow_id"],
             {"port": port.device, "dsmr_version": "2.2"},
         )
+        await hass.async_block_till_done()
```

Comment:
We don't need this here as we're getting an error in the form. Then there are no more possible tasks.
```suggestion
```